api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -141,13 +142,6 @@ message Cluster {
   // this setting is
   // ignored.
   // [#next-major-version: deprecate AUTO in favor of a V6_PREFERRED option.]
-  enum DnsLookupFamily {
```

Comment:
We can't remove existing enums or messages, since that would be a breaking change for generated code using these protos.

Instead, we'll have to duplicate these messages in the new file.  The old versions in this file will be deprecated, and we'll advise people to use the new ones.